lib/graph.py

```diff
@@ -34,6 +34,11 @@ def escape(string):
         return string
 
 
+def wrap_command(limit, break_with, command):
+    occurrences = re.findall('.{1,%i}' % limit, command)
+    newline_separated_command = break_with.join(occurrences)
+    return newline_separated_command
+
 
 class DependencyNode(Node):
     def __init__(self, fyle, truncate=30, **style):
```

Comment:
Suggest changing the `truncate` argument to `line_width`, since we're no longer truncating.